python/src/trezorlib/cli/benfen.py

```
@@ -0,0 +1,90 @@
+
+
+@click.group(name="benfen")
+def cli():
+    """Benfen commands."""
+
+
+@cli.command()
+@click.option("-n", "--address", required=True, help=PATH_HELP)
+@click.option("-d", "--show-display", is_flag=True)
+@with_client
+def get_address(client: "TrezorClient", address: str, show_display: bool) -> str:
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)